addons/api/mirage/factories/channel-recording.js

```diff
@@ -16,6 +16,7 @@ export default factory.extend({
       : faker.date.recent({ days: 3, refDate: this.end_time });
   },
   end_time: (i) => (i % 3 === 2 ? null : faker.date.recent()),
-  duration: (i) => (i % 3 === 2 ? null : `${faker.number.int()}s`),
+  duration: (i) =>
+    i % 3 === 2 ? null : `${faker.number.int({ max: 100000 })}s`,
```

Comment:
I went with `100,000` seconds because that puts us just over 27 hours which can give us the full spectrum of seeing `Xd Xh Xm Xs`.